randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -275,7 +275,7 @@ Extra - asset_id: collision_camera_003
           Morph Ball
           Any of the following:
               Flash Shift or Grapple Beam or Space Jump or Speed Booster
-              # Bomb horizontally to reach the item
+              # HBJ: https://youtu.be/8T5vedkSQRM?si=3myhX10arnQOZXF-
```

Comment:
This comment should probably not use the `HBJ` acronym, since the trick is called `Diagonal Bomb Jump` in this case. Also, full disclosure, I'm the one who changed this from an IBJ to a DBJ, and I'm not completely sure how I feel about it.

Reply:
I feel like "Diagonal Bomb Jump" is too much text and starts to clutter the window. Is "DBJ" acceptable or is it better to just have the link with no text to guide the reader?

Reply:
DBJ is probably fine here since it's right next to the fully spelled out version. When these are clearly connected to a specific part of the logic structure, I usually just put it in as only the link, without specifying what it's referring to, though I'm not opposed to doing it.